backend/services/grading_embedding_service.py

```
@@ -0,0 +1,388 @@
+                    'chunk_index': chunk.chunk_index
+                }
+                for chunk in chunks
+            ]
+            
+        except Exception as e:
+            debug_log(f"[GRADING_RETRIEVAL] Error getting simulation chunks: {str(e)}")
+            return []
+        finally:
+            db.close()
+    
+    def _update_material_status(
```

Ruff (0.14.1)

Missing return type annotation for private function `_update_material_status`

Add return type annotation: `None`

(ANN202)